Libs/Image/Image.cpp

```diff
@@ -726,16 +753,18 @@ Image& Image::gaussianBlur(double sigma)
   return *this;
 }
 
-Image& Image::crop(const Region &region)
+Image& Image::crop(PhysicalRegion region, const int padding)
 {
+  region.shrink(physicalBoundingBox()); // clip region to fit inside image
   if (!region.valid())
-    std::cerr << "Invalid region specified." << std::endl;
+    std::cerr << "Invalid region specified (it may lie outside physical bounds of image)." << std::endl;
```

Comment:
Does this need to return?  Or throw an exception?  What happens in the ITK filter below if the region is invalid?

Reply:
The itk filter takes a zero-based index region, but we wanted the user to consider the physical region, which is consistent with Mesh. If we decided to also have an index region we can add this at any time.